app/main.py

```diff
@@ -1,52 +1,42 @@
-
-
-class Book:
-    def __init__(self, title: str, content: str):
-        self.title = title
-        self.content = content
-
-    def display(self, display_type: str) -> None:
-        if display_type == "console":
-            print(self.content)
-        elif display_type == "reverse":
-            print(self.content[::-1])
-        else:
-            raise ValueError(f"Unknown display type: {display_type}")
-
-    def print_book(self, print_type: str) -> None:
-        if print_type == "console":
-            print(f"Printing the book: {self.title}...")
-            print(self.content)
-        elif print_type == "reverse":
-            print(f"Printing the book in reverse: {self.title}...")
-            print(self.content[::-1])
-        else:
-            raise ValueError(f"Unknown print type: {print_type}")
-
-    def serialize(self, serialize_type: str) -> str:
-        if serialize_type == "json":
-            return json.dumps({"title": self.title, "content": self.content})
-        elif serialize_type == "xml":
-            root = ET.Element("book")
-            title = ET.SubElement(root, "title")
-            title.text = self.title
-            content = ET.SubElement(root, "content")
-            content.text = self.content
-            return ET.tostring(root, encoding="unicode")
-        else:
-            raise ValueError(f"Unknown serialize type: {serialize_type}")
+from book.book import (Book,
+                       ConsoleBookPrinter,
+                       ReverseBookPrinter,
+                       DisplayConsole,
+                       ReversDisplayConsole,
+                       )
+from serializers.serializers import JSONBookSerializer, XMLBookSerializer
 
 
 def main(book: Book, commands: list[tuple[str, str]]) -> None | str:
     for cmd, method_type in commands:
         if cmd == "display":
-            book.display(method_type)
+            if method_type == "console":
+                book_printer = DisplayConsole()
+            elif method_type == "reverse":
+                book_printer = ReversDisplayConsole()
+            else:
+                raise ValueError(f"Unknown display type: {method_type}")
+
+            book_printer.display_book(book)
+
         elif cmd == "print":
-            book.print_book(method_type)
+            if method_type == "console":
+                book_printer = ConsoleBookPrinter()
+            elif method_type == "reverse":
+                book_printer = ReverseBookPrinter()
+            else:
+                raise ValueError(f"Unknown display type: {method_type}")
+
+            book_printer.print_book(book)
+
         elif cmd == "serialize":
-            return book.serialize(method_type)
+            if method_type == "json":
+                book_serializer = JSONBookSerializer()
+            elif method_type == "xml":
+                book_serializer = XMLBookSerializer()
+            else:
+                raise ValueError(f"Unknown serialize type: {method_type}")
```

Comment:
This approach is not good because when adding new command or method you should add new `if` statement and it violates open-closed principle. Refactor it